innovation.game.php

```diff
@@ -6667,6 +6667,13 @@ function claimSpecialAchievement($player_id, $achievement_id) {
             self::transferCardFromTo($achievement, $player_id, 'achievements');
         }
     }
+    
+    function getAchievementEligibility($player_id, $age) {
+        $cards = self::getCardsInLocationKeyedByAge($player_id, 'achievements');
+        
+        return (self::getPlayerScore($player_id) > (5 * count($cards[$age] + 1)));
+    }
```

Comment:
I'm deleting `getAchievementEligibility` since it was unused and it also has a bug. You're multiplying by 5 *before* adding 1. You need to add 1 before multiplying.